Cargo.toml

```diff
@@ -20,17 +20,17 @@ codecov = { repository = "jonhoo/flurry", branch = "master", service = "github"
 maintenance = { status = "experimental" }
 
 [features]
-sanitize = ['crossbeam-epoch/sanitize']
+#sanitize = ['crossbeam-epoch/sanitize']
```

Comment:
Did this feature actually get removed, or just renamed? I forget whether we make use of `sanitize` directly in the code of this crate, but if we do this should now be
```toml
sanitize = []
```
If we don't, the feature should be removed entirely, and you'll also need to update the CI config to not try to run with `--features sanitize`.

Reply:
Yes, the feature is removed by crossbeam-epoch.